gatsby-theme-landing-page/src/sections/features.js

```diff
@@ -37,14 +37,20 @@ function Feature({
 
   return (
     <div className={`${styles.featureContainer} ${orientationStyle}`}>
-      <div className={styles.copyContainer}>
-        <MarkdownText as="h4" className={styles.primaryText} {...primaryText} />
-        <MarkdownText className={styles.secondaryText} {...secondaryText} />
-        <LinkContainer>
-          {links && links.map((link) => <Link key={link.id} {...link} />)}
-        </LinkContainer>
+      <div className={styles.copyColumn}>
+        <div className={styles.copyContainer}>
```

Comment:
Why do we need the nested div here? Not seeing anything obvious in the css

Reply:
I want the column to take up 50% width, but contain the copy itself to 512px.